src/pages/[platform]/deploy-and-host/fullstack-branching/monorepos/index.mdx

```diff
@@ -75,8 +75,43 @@ The first app, `my-shared-backend`, will be the only app that updates changes to
 
 1. For the frontend apps, connect the frontend projects in the Amplify console separately, and update the build commands to include:
 
-```
+```bash showLineNumbers={false}
 npx amplify generate config --branch main --app-id BACKEND-APP
 ```
 
 - `BACKEND-APP` is the `app-id` for your backend app. You can find this by going to the Amplify console and navigating to _backend-app > App settings > App ARN_ and copying the ID located at the end of the string `arn:aws:amplify:<region>:<account-id>:apps/app-id`.
+
+You can run the following one of the following commands in the terminal to generate client code that is specific to your needs:
+
+
+<BlockSwitcher>
+<Block name="Flutter">
+```bash showLineNumbers={false}
+npx amplify generate config --app-id=<app-id> --branch=main --format=dart --out-dir=lib
+```
+</Block>
```

Comment:
@reesscot for next-release/main, should we use `InlineFilter` instead?

Reply:
As per my discussion with @hbuchel as well. I will use InlineFilter instead.

Reply:
Done ✅